data/io.elementary.terminal.gschema.xml

```diff
@@ -44,8 +44,13 @@
     </key>
     <key name="zoom" type="d">
       <default>1.0</default>
-      <summary>Most recent zoom level of terminal</summary>
-      <description>Most recent zoom level of terminal</description>
+      <summary>Default zoom level of new terminal</summary>
+      <description>Default zoom level of new terminal</description>
+    </key>
+    <key name="tab-zooms" type="as">
+      <default>[]</default>
```

Comment:
```suggestion
      <default>['1.0']</default>
```

Reply:
Hmm, that is not the intended behaviour - there is already a separate "zoom" setting for the fallback zoom for new tabs.  I expected that to be used where a tab did not have a corresponding "tab-zoom" value (which cannot guaranteed as the settings could be manually sent to an empty or invalid array regardless of the default value).  I'll look at this part of the code more closely.

Reply:
OK, recent commits should fix the problem.